IRremoteESP8266.h

```diff
@@ -232,18 +232,19 @@ class IRsend
   void sendKelvinator(unsigned char data[]);
   void sendSherwood(unsigned long data, int nbits=32, unsigned int repeat=1);
   void sendMitsubishiAC(unsigned char data[]);
-  void enableIROut(int khz);
+  void enableIROut(unsigned int khz, uint8_t duty=50);
```

Comment:
Using `void enableIROut(unsigned int khz, uint8_t duty=50);` means that it can be invoked as `enableIROut(38);` so it should be backward compatible. i.e. duty will default to _50_ if it isn't specified. Thus I don't think we need to use an overload. I'm not a C++ expert by any means, so if there is a significant difference, can you explain it to me, or point me to some documentation?
I've been using this _feature_ for a number of routines and it's worked fine so far. But, I'm always happy to learn & change my ways.